bin/update_asset

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+require File.expand_path("../../config/environment", __FILE__)
+
+filename = ARGV.any? ? ARGV.fetch(0) : nil
+old_asset_id = ARGV.any? ? ARGV.fetch(1) : nil
+
+unless filename
+  puts "You need to provide a filename as first argument when running this script"
+  abort
+end
+
+unless old_asset_id
+  puts "You need to provide the old ID path as second argument when running this script"
```

Comment:
I think it would be better to refer to this as the "asset ID" instead of the "old ID path"